libs/blocks/section-metadata/section-metadata.js

```diff
@@ -1,17 +1,32 @@
 function handleBackground(div, section) {
-  const pic = div.querySelector('picture');
+  const pic = div.background.content.querySelector('picture');
   if (pic) {
     section.classList.add('has-background');
     pic.classList.add('section-background');
+    handleFocalpoint(pic, div.background.content);
     section.insertAdjacentElement('afterbegin', pic);
   } else {
-    const color = div.textContent;
+    const color = div.background.content.textContent;
     if (color) {
       section.style.background = color;
     }
   }
 }
 
+export function handleFocalpoint(pic, child) {
+  if (!(child)) return;
+  let text = '';
+  if (child.childElementCount == 2) {
+    text = child.querySelectorAll('p')[1].textContent;
+  } else if (child.textContent) {
+    text = child.textContent;
+  }
+  const image = pic.querySelector('img');
+  const directions = text.slice(text.indexOf(':') + 1).split(',');
+  const [x,y = ''] = directions
+  image.style.objectPosition = `${x.trim().toLowerCase()} ${y.trim().toLowerCase()}`;
```

Comment:
nit: might be nice to check for the image here as well:
 `if (image) image.style.objectPosition = '${x.trim().toLowerCase()} ${y.trim().toLowerCase()}';`

Reply:
done